lldb/source/Interpreter/ScriptInterpreter.cpp

```diff
@@ -150,6 +150,11 @@ ScriptInterpreter::GetOpaqueTypeFromSBExecutionContext(
   return exe_ctx.m_exe_ctx_sp;
 }
 
+lldb::StackFrameListSP ScriptInterpreter::GetOpaqueTypeFromSBFrameList(
```

Comment:
OpaqueType is a confusing name for the underlying StackFrameList.  GetStackFrameListFromSBFrameList seems much clearer.  Also, why isn't this a method on SBFrameList?

Reply:
1. This is how we've been doing it. Renaming these methods would be orthogonal to this PR.
2. We can't have it be in SBFrameList because each ScriptingInterface will have to talk to SBAPI classes directly would be a layering violation. We had to use the ScriptInterpreter for that.